app/views/budgets/_budget_card.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <table class="table table-striped", style="text-align: center;">
+    <thead>
+      <tr>
+        <th>No.</th>
+        <th>Name</th>
+        <th>Amount</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @group.budgets.each_with_index do |budget, index| %>
+        <tr>
+          <td><%= index + 1 %></td>
+          <td><%= budget.name %></td>
+          <td>$ <%= budget.amount %></td>
+          </td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+    </div>
+</div>
```

Comment:
- [ ] I would suggest keeping the design same.

## Your design
![Screenshot from 2022-10-22 02-09-41](https://user-images.githubusercontent.com/26767383/197285165-ece44b48-197d-4c74-a941-fe4a4dc628ad.png)


## Given design
![Screenshot from 2022-10-22 01-53-34](https://user-images.githubusercontent.com/26767383/197285001-ad56a954-a26b-4a0a-bdbb-512fb3f5fb1e.png)

Reply:
**Thank You Sir I brought the Changes**

![Screenshot (123)](https://user-images.githubusercontent.com/98980908/197322003-a59665c6-ba54-4880-901d-42a9d8b13937.png)